webapp/TargetedMS/js/QCTrendPlotPanel.js

```diff
@@ -179,7 +179,7 @@ Ext4.define('LABKEY.targetedms.QCTrendPlotPanel', {
 
     calculateStartDateByOffset : function() {
         if (this.dateRangeOffset > 0) {
-            var todayMinusOffset = new Date();
+            var todayMinusOffset = this.maxAcquiredTime ? new Date(this.maxAcquiredTime) : new Date();
```

Comment:
should probably rename this variable now: `startDateByOffset`?

Reply:
Renamed